spec/system/admin_reward_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'AdminReward crud', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:admin) { create(:admin_user) }
+
+  it 'test admin crud actions' do
+    visit admin_root_path
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: admin.password
+    click_button 'Log in'
+    expect(page).to have_content 'Admin dashboard'
```

Comment:
You are already checking this in another spec - consider using login helpers:
https://sajadtorkamani.com/docs/rails-testing/login-devise-user-in-capybara-tests.html

Reply:
Thanks for the hint.